### DISCUSSION TITLE
Welcome to DevSecOps-Playbook Discussions!

### DISCUSSION THREAD

**Discussion by 6mile** [6mile/DevSecOps-Playbook#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Ken12R** [comment]
Hi there, I am looking for a resource like a NIST framework or another resource that shows that we need to have segregation of duties in DevSecOps in the context of developing code / apps and analysis them. I know that it needs to done by different team. Can you please point to any resource that explains more ?
